### PR TITLE
fix excluding rules

### PR DESCRIPTION
**Pull request by bhirsz**

### CODE REVIEW

File: robocop/config/config.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def is_rule_enabled(self, msg):
             if msg.msg_id not in self.include and msg.name not in self.include:
                 return False
         if self.exclude:
-            if msg.msg_id in self.include or msg.name in self.include:
+            if msg.msg_id in self.exclude or msg.name in self.exclude:
```
Comment (bhirsz) [inline robocop/config/config.py:59]: This show how proper testing is important :D @mnojek

File: robocop/config/config.py [hunk 2]
```diff
@@ -23,8 +23,8 @@ def __init__(self):
         self.configure = list()
         self.format = "{source}:{line}:{col} [{severity}] {msg_id} {desc}"
         self.paths = []
-        self.include_patterns = None
-        self.exclude_patterns = None
+        self.include_patterns = list()
```
Comment (mnojek) [inline robocop/config/config.py:26]: Why not to just create empty list by calling `self.include_patterns = []`? I think it's more concise and even a bit faster.

File: robocop/config/config.py [hunk 3]
```diff
@@ -55,15 +55,12 @@ def is_rule_enabled(self, msg):
         if self.include and not self.include_patterns:
             if msg.msg_id not in self.include and msg.name not in self.include:
                 return False
-        if self.exclude:
-            if msg.msg_id in self.include or msg.name in self.include:
+        if msg.msg_id in self.exclude or msg.name in self.exclude:
```
Comment (mnojek) [inline robocop/config/config.py:58]: You can also type 
```python
if any(x in [msg.msg_id, msg.name] for x in self.exclude):
```
but I am not sure if it is more readable.

Comment (bhirsz) [reply robocop/config/config.py:58]: actually in this case self.exclude is set so it's faster to check if msg is in self.exclude that other way around. 

Your code would be then this:
```
any(m in self.exclude for m in (msg.msg_id, msg.name)
```
But I think just two ors is bit more readable than that

**Pull request merged**